user-guide/Advanced_Modules/DOM/DOM_best_practices.md

```diff
@@ -0,0 +1,200 @@
+
+## DOM model
+
+### Try to keep the DomInstance objects small
+
+When interacting with a `DomInstance`, all data for this instance needs to be retrieved and/or sent to the server. Therefore, less data results in faster actions. It is advisable to think critically and limit the size of `DomInstances`, following these recommendations:
+
+#### Only include data that is actually required
+
+When designing a DOM model, there is a tendency to include any possible piece of information relevant to the represented entity. It is advised to include only the needed data to ensure that `DomInstances` are kept small and space-efficient. Updating the DOM model with an extra field later on is much easier than removing one, which was not needed in hindsight.
+
+#### Avoid data duplication
+
+Always strive to store certain data only once and link to it where needed. This not only benefits the size of `DomInstances` but also makes updating this data more straightforward, as it only needs to be done in one location.
+
+#### Avoid storing JSON in a string field
+
+While it may be tempting to model JSON and store it in a string field, this is considered bad practice. It hinders data checks, makes the model hard to understand, and introduces a higher risk of storing irrelevant data, leading to performance issues. It is recommended to use multiple sections or other `DomInstances` when possible.
+
+#### Limit the amount of sections/fields
+
+Allowing multiple sections for a `SectionDefinition` theoretically enables the addition of an unlimited number of sections to the `DomInstance`. However, every section increases the `DomInstance`'s size, impacting performance, especially when each section contains numerous fields. For such `SectionDefinitions`, it is crucial to ensure that only the most important values are stored.
+
+### Store rarely used (meta) data in separate DomInstances
+
+In most solutions, there are objects that need to be retrieved quite often to display them in a table or on a timeline.  It is recommended to keep these often-retrieved `DomInstances` as small as possible and include only the required data. If values are only needed when a specific object is known (e.g., selected in an LCA table), create a separate metadata object that stores these additional values. A link can then be made to retrieve these values only when needed. However, be cautious, as this introduces another read call, so ensure that these values are needed less often than the main object values. If these additional values will always be joined to the main object, the performance benefit may be negated or even have the opposite effect.
```

Comment:
S: the performance benefit may be **negligible**

Reply:
With negated, I mean to convey that the performance benefit is nullified, so that there is no point in doing this when using the joins.